algorithm/overlay/sweepline/sweep_line_index.go

```diff
@@ -0,0 +1,83 @@
+//Package sweepline Contains struct which implement a sweepline algorithm for scanning geometric data structures.
+package sweepline
```

Comment:
`sweep_line_index` should be in `index` package

Reply:
It shouldn't be in index package，it has not implemented index interface.